lib/SelectedObjectsProvider/useObjectSelector.js

```diff
@@ -0,0 +1,49 @@
+import { useContext } from 'react';
+import { SelectedObjectsContext } from './index';
+
+export function useObjectSelector(
+  id,
+  type,
+  idsToSelect,
+  disabledCallback = () => false
```

Comment:
disabledCallback makes it seem like it fires when a object is disabled. Maybe `isDisabledChecker` or something might be better?

Reply:
👍